include/picongpu/plugins/binning/BinningData.hpp

```diff
@@ -32,6 +32,15 @@ namespace picongpu
 {
     namespace plugins::binning
     {
+        /** @brief Bin particles in enabled region */
+        enum ParticleRegion : uint32_t
+        {
+            /** Bounded - Particles inside the simulation volume */
+            Bounded = 1 << 0, // 01 in binary, corresponds to the first bit
```

Comment:
please move the comment above to avoid auto formation issues

Reply:
That the options must be represendet by unique bit should be a comment in the doc string of ParticleRegion

Reply:
moved